From 1da0dbfc23cf41cc1dccb7829805032a46c17c1d Mon Sep 17 00:00:00 2001
From: Stefan Assmann <sassmann@redhat.com>
Date: Tue, 2 Sep 2014 11:39:09 -0400
Subject: [ethernet] i40e: debugfs fix to dump remote LLDPDU

Message-id: <1409658042-22689-189-git-send-email-sassmann@redhat.com>
Patchwork-id: 90380
O-Subject: [RHEL7.1 PATCH 188/281] i40e: debugfs fix to dump remote LLDPDU
Bugzilla: 1091126
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Neerav Parikh <neerav.parikh@intel.com>

Fix the debugfs command "lldp get remote" that dumped the local
LLDPDU instead of peer's LLDPDU.

Change-ID: I0702eacdafd54478c18f20cab3a7fa5dc1b3182d
Signed-off-by: Neerav Parikh <neerav.parikh@intel.com>
Signed-off-by: Jeff Kirsher <jeffrey.t.kirsher@intel.com>
Signed-off-by: Stefan Assmann <sassmann@redhat.com>
(cherry picked from commit c27936e7b2d57af2965a78973430383be98fe88a)
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/net/ethernet/intel/i40e/i40e_debugfs.c b/drivers/net/ethernet/intel/i40e/i40e_debugfs.c
index cffdfc2..910b01b 100644
--- a/drivers/net/ethernet/intel/i40e/i40e_debugfs.c
+++ b/drivers/net/ethernet/intel/i40e/i40e_debugfs.c
@@ -1830,7 +1830,7 @@ static ssize_t i40e_dbg_command_write(struct file *filp,
 
    ret = i40e_aq_get_lldp_mib(&pf->hw,
      I40E_AQ_LLDP_BRIDGE_TYPE_NEAREST_BRIDGE,
-     I40E_AQ_LLDP_MIB_LOCAL,
+     I40E_AQ_LLDP_MIB_REMOTE,
      buff, I40E_LLDPDU_SIZE,
      &llen, &rlen, NULL);
    if (ret) {
-- 
1.7.1